server/datastore/mysql/migrations/tables/20220908181826_AddOrbitNodeKeyToHosts.go

```diff
@@ -0,0 +1,18 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220908181826, Down_20220908181826)
+}
+
+func Up_20220908181826(tx *sql.Tx) error {
```

Comment:
we generally add tests for the migrations too

Reply:
thanks, added